### ISSUE TITLE
Show band leaders which of their members are not yet active

### ISSUE THREAD

**Issue by bklang** [Gig-o-Matic/GO3#272]
The intent is to help band leaders identify which of their members have not yet migrated to the new Gig-O. Put a badge next to each member's name who does not have a password created yet.

**Comment by Flite999** [comment]
👀

**Comment by aaronoppenheimer** [comment]
Do we still need this, since the new gig-o is up and running?

**Comment by Flite999** [comment]
Have all the users been migrated?

**Comment by aaronoppenheimer** [comment]
Yes! Kind of by definition - the old gig-o shut down in June and everyone's on the new one.

**Comment by Flite999** [comment]
Ok! I'll look for another issue to contribute to.
